packages/shared-base/src/Persona/type.ts

```diff
@@ -11,11 +11,14 @@ export interface PersonaInformation {
     linkedProfiles: ProfileInformation[]
 }
 export interface ProfileInformation {
+    [index: string]: unknown
```

Comment:
DON'T DO THIS. It is NOT a record. What use case do you need to add an indexed signature on it?

Reply:
deleted